### PR TITLE
wallpaperctl: new, 0.1.3

### PR DESCRIPTION
**Pull request by neruthes**
Topic Description
-----------------

Add packages:

- `wallpaperctl`.

Package(s) Affected
-------------------

- wallpaperctl

Security Update?
----------------

- [ ] Yes
- [x] No

Architectural Progress
----------------------

- [ ] Architecture-independent `noarch`

### CODE REVIEW

File: extra-utils/wallpaperctl/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,3 @@
+abinfo "Building \"wallpaperctl\"..."
+
+install -Dvm755 wallpaperctl "$PKGDIR"/usr/bin/wallpaperctl
```
Comment (MingcongBai) [inline extra-utils/wallpaperctl/autobuild/build]: Please use absolute path here, i.e. `"$SRCDIR"/wallpaperctl`.

File: extra-utils/wallpaperctl/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=wallpaperctl
+PKGSEC=utils
+PKGDEP="bash x11-app feh grep sed"
+BUILDDEPONLY=1
```
Comment (MingcongBai) [inline extra-utils/wallpaperctl/autobuild/defines]: Variable not yet implemented in Autobuild3. Please remove this line.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-utils/wallpaperctl/autobuild/build (hunk 1 above)
Comment (neruthes) [reply extra-utils/wallpaperctl/autobuild/build]: Updated.

File: extra-utils/wallpaperctl/autobuild/defines (hunk 2 above)
Comment (neruthes) [reply extra-utils/wallpaperctl/autobuild/defines]: Updated.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**